charts/juicefs-csi-driver/templates/serviceaccount.yaml

```diff
@@ -72,13 +72,36 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+    - ""
+  resources:
+    - pods
```

Comment:
Do we need cluster scope full access to pods? Can we limit it in specific namespace?